docs/docs/orgs/README.md

```diff
@@ -38,7 +38,19 @@ When you [create an org](#creating-an-org), you'll see that org listed in the co
 
 ## Migrating workflows from your user to your org
 
-If you are running resources in your user account, but want to move those to your organization, [reach out to the Pipedream team](https://pipedream.com/support).
+Migrate your account's workflows, sources and connected accounts from your account to your organization by opening your [account settings](https://pipedream.com/settings/account).
+
+Under the **User Account Settings** area, open the **Transfer Ownership** dropdown and click the **Transfer your Account** button.
+
+![Finding the resource migration tool](https://res.cloudinary.com/pipedreamin/image/upload/v1658170705/docs/components/CleanShot_2022-07-18_at_14.57.15_2x_nrxj6w.png)
+
+You'll be prompted to choose the organization you'd like to transfer your account's resources to.
+
+_All_ of your accounts workflows, sources and connected accounts will be transfered to the organization. It's not possible to migrate a specific workflow, source or connected account.
```

Comment:
Maybe here clarify that "_All_ your account's resources, including workflows, sources, and connected accounts" to clarify that it's not just those resources being moved.